test/morphometrics/test_compute_morphometrics.py

```diff
@@ -189,7 +194,7 @@ def test_get_axon_morphometrics_with_invalid_gratio_with_axon_as_ellipse(self):
             im_myelin=self.bad_pred_myelin,
             axon_shape=self.axon_shape
             )
-        assert np.isnan(stats_array[0]['gratio'])
+        assert stats_array[0]['gratio']==0
```

Comment:
Same question as before regarding NAN vs. zero, I can't remember why we are changing this.